docs/source/containerization/kubernetes.mdx

```diff
@@ -5,9 +5,206 @@ description: Using router images with kubernetes
 
 import { Link } from 'gatsby';
 
+## Sample Kubernetes Configuration
+
+### Helm
+
+[Helm](https://helm.sh) is the package manager for kubernetes.
+
+There is a complete [helm chart definition](https://github.com/apollographql/router/tree/main/helm/chart/router) in the repo which illustrates how to use helm to deploy the router in kubernetes.
+
+Here's an example which would use helm to install the router:
+ - into namespace "router-deploy" (create namespace if it doesn't exist)
+ - with helm install name "router-test"
+ - with support for prometheus enabled
+
+You would run this command from <repo>/helm/chart directory.
```

Comment:
`<repo>` is causing the docs to fail

Reply:
fixed in: 247d6f1a36bf8297acbfccc820f09c9c85517a15